docs/config.md

```diff
@@ -0,0 +1,111 @@
+# Configuring Sift Fraud Detection
+
+Sift uses machine learning and real-time data analysis to detect fraud. You can use Sift fraud detection with WSO2 Identity Server by following the steps below.
+
+## Prerequisites
+- You need to have a Sift account. If you do not have an account, create one by visiting the [Sift website](https://sift.com/).
+
+## Install the Sift connector
+
+**Step 1:** Extract the project artifacts
+1. Clone the `identity-fraud-sift-int` repository.
+2. Build the project by running the ```mvn clean install``` command in the root directory.
```

Comment:
Since we are publishing github releases lets point to downloading from the git release or the connector store and omit the building part.